polyply/tests/test_data/library_tests/martini3/P3HT/polyply/P3HT.itp

```diff
@@ -1,47 +1,47 @@
-; /Users/riccardo/miniconda3/envs/polyply/bin/polyply gen_itp -lib martini3 -name P3HT -seq P3HT:6 -o P3HT.itp
+; /coarse/fabian/current-projects/polymer_itp_builder/vermouth_dev/venvv/bin/polyply gen_params -lib martini3 -seq P3HT:6 -o P3HT.itp -name P3HT
 
 ; Please cite the following papers:
 
 [ moleculetype ]
 P3HT 1
 
 [ atoms ]
- 1 TC6 1 P3HT S1  1 0.0    
- 2 TC5 1 P3HT C2  2 0.0    
- 3 TC5 1 P3HT C3  3 0.0    
- 4 U   1 P3HT V4  4 0.0 0.0
- 5 SC2 1 P3HT C5  5 0.0    
- 6 SC2 1 P3HT C6  6 0.0    
- 7 TC6 2 P3HT S1  7 0.0    
- 8 TC5 2 P3HT C2  8 0.0    
- 9 TC5 2 P3HT C3  9 0.0    
-10 U   2 P3HT V4 10 0.0 0.0
-11 SC2 2 P3HT C5 11 0.0    
-12 SC2 2 P3HT C6 12 0.0    
-13 TC6 3 P3HT S1 13 0.0    
-14 TC5 3 P3HT C2 14 0.0    
-15 TC5 3 P3HT C3 15 0.0    
-16 U   3 P3HT V4 16 0.0 0.0
-17 SC2 3 P3HT C5 17 0.0    
-18 SC2 3 P3HT C6 18 0.0    
-19 TC6 4 P3HT S1 19 0.0    
-20 TC5 4 P3HT C2 20 0.0    
-21 TC5 4 P3HT C3 21 0.0    
-22 U   4 P3HT V4 22 0.0 0.0
-23 SC2 4 P3HT C5 23 0.0    
-24 SC2 4 P3HT C6 24 0.0    
-25 TC6 5 P3HT S1 25 0.0    
-26 TC5 5 P3HT C2 26 0.0    
-27 TC5 5 P3HT C3 27 0.0    
-28 U   5 P3HT V4 28 0.0 0.0
-29 SC2 5 P3HT C5 29 0.0    
-30 SC2 5 P3HT C6 30 0.0    
-31 TC6 6 P3HT S1 31 0.0    
-32 TC5 6 P3HT C2 32 0.0    
-33 TC5 6 P3HT C3 33 0.0    
-34 U   6 P3HT V4 34 0.0 0.0
-35 SC2 6 P3HT C5 35 0.0    
-36 SC2 6 P3HT C6 36 0.0    
+ 1 TC6 1 P3HT S1  1 0.0 36.0
```

Comment:
so, although masses in the `polyply/data/martini3/P3HT.martini3.ff` are without zero, then they get printed as floats by polyply, I see

Reply:
yep this is due to the formatting the vermouth itp-write does.